app/main.py

```diff
@@ -1,8 +1,88 @@
+from typing import List
+
+
 class Car:
-    # write your code here
-    pass
+    def __init__(self,
+                 comfort_class: int,
+                 clean_mark: int,
+                 brand: str) -> None:
+        self.comfort_class = self.validate_comfort_class(comfort_class)
+        self.clean_mark = self.validate_clean_mark(clean_mark)
+        self.brand = brand
+
+    @staticmethod
+    def validate_comfort_class(comfort_class: int) -> int:
+        if 1 <= comfort_class <= 7:
+            return comfort_class
+        return 3
+
+    @staticmethod
+    def validate_clean_mark(clean_mark: int) -> int:
+        if 1 <= clean_mark <= 10:
+            return clean_mark
+        return 5
```

Comment:
Basically you use methods wrong. You can note that all of them have the same pattern. It means you can create 1 method that accepts all parameters and return correct values. In this way you can create 1 method instead of 20 when you have 20 parameters that should be checked

Reply:
Also no need to use staticmethod because you call this method using self